src/main/java/picocli/CommandLine.java

```diff
@@ -8166,6 +8177,7 @@ private static String describe(ArgSpec argSpec, String separator, String value)
                     description = option.description();
                     descriptionKey = option.descriptionKey();
                     splitRegex = option.split();
+                    usageSplitRegex = option.usageSplit().isEmpty() ? splitRegex : option.usageSplit();
```

Comment:
Let's just store the annotation value (and we can decide later whether to use `splitRegex` or `usageSplitRegex`):

```
usageSplitRegex = option.usageSplit();
```